resources/analysisTools/rnaseqworkflow/featureCounts_2_FpkmTpm

```diff
@@ -78,8 +78,17 @@ if (length $gtf_file > 0) {
     chomp $line;
     my @line_split = split ("\t", $line);
     if ($line_split[2] eq "gene") {
-      my @comment = split ("\"",$line_split[8]);
-      $gtf{$comment[1]}="$line_split[0]\t$line_split[3]\t$line_split[4]\t$comment[1]\t.\t$line_split[6]\t$comment[9]";
+      my @comment = split (";",$line_split[8]);
+      my ($gene_id, $gene_name);
+      for(my $i=0;$i<=$#comment;$i++){
```

Comment:
Hmm. Not ideal. Another repetition in another script. This should usually be implemented only once and the code loaded as module into the Perl code.

Reply:
I didn't want to disturb the codebase. Now I have added a new subroutine to parse the GTF files via a helper package.